docs/game/cheat_sheet.md

```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
 Manners
 -------
 
-Some general manners guidelines for panel games, and specifically Just A Minute
+Panel games, much like the greatness of Great Britain, are founded on
+good manners. Here are some guidelines that make Just A Minute flow smoothly.
```

Comment:
How about:

```
Panel games, like so many things in life, are founded on good manners.
```

Or perhaps I should just remove it. 

I initially found it a bit heavy handed that a cheat sheet starts with a guide to good manners, but it's actually a really useful and important section. Lightening the tone is hard.